src/redux/cards/types.ts

```diff
@@ -2,13 +2,20 @@ import { IAttachment } from '../attachments/types'
 import { IComment } from '../comments/types'
 import { ITasksList } from '../taskLists/types'
 import { ITag } from '../tags/types'
+import { IUser } from '../users/types'
+import { IList } from '../lists/types'
 
 export type ICard = {
     id: number,
-    description: string, 
-    dueTime: Date,
-    attachments: IAttachment[],
-    comments: IComment[],
-    tasksLists: ITasksList[],
-    tags: ITag[]
+    name: string,
```

Comment:
Many problems here... some loops seems weird.

You should redo some types

Reply:
Changed.